demo/pages/index.js

```diff
@@ -1,128 +1,156 @@
+        <li>
+          <Link href="/static/[id]" as="/static/123456789">
+            <a>Static NextJS page with dynamic routing: /static/:id</a>
           </Link>
         </li>
-      ))}
-    </ul>
-
-    <h1>3. Catch-All Routes? Included ✔</h1>
-    <p>
-      You can even take advantage of{' '}
-      <a href="https://nextjs.org/docs/routing/dynamic-routes#catch-all-routes">NextJS catch-all routes feature</a>
-      .
-      <br />
-      Here are three examples:
-    </p>
-    <ul>
-      <li>
-        <Link href="/shows/[...params]" as={`/shows/73/whatever/path/you/want`}>
-          <a>/shows/73/whatever/path/you/want</a>
-        </Link>
-      </li>
-      <li>
-        <Link href="/shows/[...params]" as={`/shows/94/whatever/path/you`}>
-          <a>/shows/94/whatever/path/you</a>
-        </Link>
-      </li>
-      <li>
-        <Link href="/shows/[...params]" as={`/shows/106/whatever/path`}>
-          <a>/shows/106/whatever/path</a>
-        </Link>
-      </li>
-    </ul>
-
-    <h1>4. Static Pages Stay Static</h1>
-    <p>
-      next-on-netlify automatically determines which pages are dynamic and which ones are static.
-      <br />
-      Only dynamic pages are server-side rendered.
-      <br />
-      Static pages are pre-rendered and served directly by Netlify&apos;s CDN.
-    </p>
-
-    <ul>
-      <li>
-        <Link href="/static">
-          <a>Static NextJS page: /static</a>
-        </Link>
-      </li>
-      <li>
-        <Link href="/static/[id]" as="/static/123456789">
-          <a>Static NextJS page with dynamic routing: /static/:id</a>
-        </Link>
-      </li>
-    </ul>
-
-    <h1>Want to Learn More?</h1>
-    <p>
-      Check out the <a href="https://github.com/FinnWoelm/next-on-netlify-demo">source code on GitHub</a>.
-    </p>
-  </div>
-)
+      </ul>
+
+      <h2>5. Localization As Expected</h2>
+      <p>
+        Localization (i18n) is supported! This demo uses <code>fr</code> with <code>en</code> as the default locale (at <code>/</code>).
+      </p>
+      <strong>The current locale is {locale}</strong>
+      <p>Click on the links below to see the above text change</p>
+      <ul data-testid="list-localization">
+        <li>
+          <Link href="/fr">
+              <a>/fr</a>
+          </Link>
+        </li>
+        <li>
+          <Link href="/en">
+              <a>/en (default locale)</a>
+          </Link>
+        </li>
+      </ul>
+      
+      <h1>Want to Learn More?</h1>
+      <p>
+        Check out the <a href="https://github.com/FinnWoelm/next-on-netlify-demo">source code on GitHub</a>.
+      </p>
+    </div>
+  )
+}
 
 Index.getInitialProps = async function () {
+  const dev = process.env.CONTEXT !== 'production';
+
   // Set a random page between 1 and 100
   const randomPage = Math.floor(Math.random() * 100) + 1
+  // FIXME: stub out in dev
+  const server = dev ? `https://api.tvmaze.com/shows?page=${randomPage}` : `https://api.tvmaze.com/shows?page=${randomPage}`;
```

Comment:
You could find the base server so you can load the stub by checking if `document` is available, and either getting it from there or otherwise seeing if you can get it from `context.req`